Sources/SecureEnclaveValet.swift

```diff
@@ -90,8 +90,8 @@ public final class SecureEnclaveValet: NSObject {
     
     // MARK: Hashable
     
-    public override var hashValue: Int {
-        return service.description.hashValue
+    public override var hash: Int {
+        return service.description.hash
```

Comment:
Actually, one more question: do we want to use `hash` on the `description`, or `hashValue`? Or does it not matter because `String` and `NSString` are bridged due to our `Foundation` import?

Reply:
Good catch! I’ll revert these back to use `hashValue` since `description` is defined as a Swift `String`.